From 43a26927319f35b29e6c7246dbe93024e1d28452 Mon Sep 17 00:00:00 2001
From: Trent Mick <trentm@gmail.com>
Date: Wed, 17 Apr 2019 11:01:43 -0700
Subject: [PATCH] joyent/node-triton#263 config.js validateProfile is confused
 about how to be insecure Reviewed by: Chris Burroughs
 <chris.burroughs@joyent.com> Approved by: Chris Burroughs
 <chris.burroughs@joyent.com>

---
 CHANGES.md    | 3 +++
 lib/config.js | 4 ++--
 2 files changed, 5 insertions(+), 2 deletions(-)

diff --git a/CHANGES.md b/CHANGES.md
index 925b040..7dcf302 100644
--- a/CHANGES.md
+++ b/CHANGES.md
@@ -6,6 +6,9 @@ Known issues:
 
 ## not yet released
 
+- [joyent/node-triton#263] Correct envvar name to `TRITON_TLS_INSECURE`
+  in CLI profile validation error message.
+
 ## 7.0.2
 
 - [TRITON-1347] Changing to node-tap as the test runner. Also some
diff --git a/lib/config.js b/lib/config.js
index ea9dc36..b5708b2 100644
--- a/lib/config.js
+++ b/lib/config.js
@@ -5,7 +5,7 @@
  */
 
 /*
- * Copyright 2017 Joyent, Inc.
+ * Copyright 2019 Joyent, Inc.
  */
 
 /*
@@ -229,7 +229,7 @@ function validateProfile(profile, profilePath) {
             profile.name === 'env' ? 'TRITON_KEY_ID or SDC_KEY_ID'
                 : 'profile.keyId');
         assert.optionalBool(profile.insecure,
-            profile.name === 'env' ? 'TRITON_INSECURE or SDC_INSECURE'
+            profile.name === 'env' ? 'TRITON_TLS_INSECURE or SDC_TLS_INSECURE'
                 : 'profile.insecure');
         assert.optionalString(profile.user,
             profile.name === 'env' ? 'TRITON_USER or SDC_USER'
-- 
2.21.0

